Generator/functions.php

```diff
@@ -413,6 +414,14 @@ function git_init() {
   if ( $config[ 'git-repo' ] === 'true' ) {
     exec( 'cd ' . getcwd() . DIRECTORY_SEPARATOR . WPBP_PLUGIN_SLUG . '; git init' );
     $clio->styleLine( '😎 .git folder generated', $white );
+    $file = getcwd() . DIRECTORY_SEPARATOR . WPBP_PLUGIN_SLUG . '/.gitignore';
+    $file_content = file_get_contents( $file );
+    $file_content = str_replace( '/plugin-name/', '', $file_content );
```

Comment:
Yes, I saw the function, but that function replace the generic plugin name with the new plugin name; but in this case the string "/plugin-name/" must be removed and not replaced with the new slug.